src/main/java/uk/nhs/digital/nhsconnect/lab/results/mesh/MeshWorkflowUnknownException.java

```diff
@@ -0,0 +1,15 @@
+package uk.nhs.digital.nhsconnect.lab.results.mesh;
+
+import lombok.Getter;
+import uk.nhs.digital.nhsconnect.lab.results.rest.exception.LabResultsBaseException;
+
+@Getter
+public class MeshWorkflowUnknownException extends LabResultsBaseException {
```

Comment:
i don't think this is used

Reply:
No it isn't being used anywhere so I've removed it